packages/snaps-controllers/src/services/AbstractExecutionService.ts

```diff
@@ -397,7 +402,7 @@ export abstract class AbstractExecutionService<WorkerType>
         params: {
           origin,
           handler,
-          request,
+          request: request as JsonRpcRequest,
```

Comment:
why is casting needed here?

Reply:
`request` is typed as `Record<string, unknown>`, but AFAIK we validate that it's a `JsonRpcRequest` before reaching this point.

Reply:
Yeah, we do, we just don't have a better type right here